.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-   - "0.10"
-
+  - "0.10"
 script:
-- "npm run jshint"
-- "npm run travis"
\ No newline at end of file
+  - "npm run jshint"
+  - "npm run travis"
+after_script: "npm install coveralls && cat ./coverage/lcov.info | coveralls"
```

Comment:
It is common to specify a specific version of coveralls to install here (again, see express), though I prefer to always use the latest. Let me know if you prefer something else here.